pygmt/helpers/decorators.py

```diff
@@ -273,7 +286,12 @@ def new_module(*args, **kwargs):
             for arg, fmt in conversions.items():
                 if arg in kwargs:
                     value = kwargs[arg]
-                    issequence = fmt in ("sequence", "sequence_comma", "sequence_plus")
+                    issequence = fmt in (
+                        "sequence",
+                        "sequence_comma",
+                        "sequence_plus",
+                        "sequence_space",
+                    )
```

Comment:
I think you can change these lines to
```python
issequence = fmt in valid_conversions
```

Reply:
Yeah, in fact I could probably refactor the code to just have a `separators` dictionary and access the `separators.keys()` instead of having valid_conversions.

Reply:
I agree, but it may be better to open another PR after this one is merged.

Reply:
Agreed. There's lots of room to refactor that function such as using f-strings since we're supporting python > 3.6 anyway.